commercial/app/controllers/commercial/JobAds.scala

```diff
@@ -11,19 +11,18 @@ object JobAds extends Controller with implicits.Requests {
 
   implicit val codec = Codec.utf_8
 
-  val jobCategories = Map(
-    "arts" -> "Arts",
-    "graduate" -> "Graduate",
-    "social-care" -> "Social care",
-    "charity" -> "Charity",
-    "health" -> "Health",
-    "higher-education" -> "Higher education",
-    "environment" -> "Environment",
-    "housing" -> "Housing",
-    "schools" -> "Schools",
-    "government" -> "Government",
-    "media" -> "Media",
-    "" -> "All sectors »"
+  val jobCategories = Seq(
+    Seq("arts", "Arts"),
+    Seq("graduate", "Graduate"),
+    Seq("social-care", "Social care"),
+    Seq("charity", "Charity"),
+    Seq("health", "Health"),
+    Seq("higher-education", "Higher education"),
+    Seq("environment", "Environment"),
+    Seq("housing", "Housing"),
+    Seq("schools", "Schools"),
+    Seq("government", "Government"),
+    Seq("media", "Media")
```

Comment:
The ordering in the `for` comprehension was not consistent with the order in the definition, that's why I used sequences instead. Unless there is a better way?